<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFlightDialog</class>
 <widget class="QDialog" name="EditFlightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>дата</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>откуда</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fromTownComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>куда</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="toTownComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>самолет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="aircraftComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>цена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="priceSpinBox">
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="applyPushButton">
         <property name="text">
          <string>сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deletePushButton">
         <property name="text">
          <string>удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClosePushButton">
         <property name="text">
          <string>отменить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
